msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -5,62 +5,59 @@ description: Identify the meeting apps API references with examples and Code sam
 ms.topic: conceptual
 ms.author: lajanuar
 ms.localizationpriority: medium
-keywords: teams apps meetings user participant role api usercontext notification signal query 
+keywords: teams apps meetings user participant role api user context notification signal query 
 ---
 
 # Meeting apps API references
 
-The meeting extensibilities provide APIs to transform the meeting experience:
+The meeting extensibility provide APIs to enhance meeting experience. You can perform the following with help of the listed APIs:
 
 * Build apps or integrate existing apps within meeting lifecycle.
-* Use the APIs to make your app aware of the meeting.
-* Select the APIs you want to use to enhance the meeting experience.
+* Use APIs to make your app aware of meeting.
+* Select required APIs to improve the meeting experience.
 
 The following table provides a list of APIs:
 
-|API|Description|Request|Source|
-|---|---|----|---|
-|**GetUserContext**| Enables you to get contextual information to display relevant content in a Teams tab. |_**microsoftTeams.getContext( ( ) => {  /*...*/ } )**_|Microsoft Teams client SDK|
-|**GetParticipant**| Enables a bot to fetch participant information by meeting ID and participant ID. |**GET** _**/v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}**_ |Microsoft Bot Framework SDK|
-|**NotificationSignal** | Enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. It allows you to signal based on user action that shows an in-meeting dialog box. |**POST** _**/v3/conversations/{conversationId}/activities**_|Microsoft Bot Framework SDK|
```

Comment:
Each of the detail sections should have a request section that should provide the request information that you are deleting from the table.

Reply:
Done, @AkJo, I am unsure about the notification signal API request, could you conform please.